policy/modules/contrib/nsd.fc

```diff
@@ -14,7 +14,7 @@
 /usr/sbin/nsd-control-setup		--	gen_context(system_u:object_r:nsd_exec_t,s0)
 
 /var/lib/nsd(/.*)?		gen_context(system_u:object_r:nsd_zone_t,s0)
-/var/run/nsd\.pid	--	gen_context(system_u:object_r:nsd_var_run_t,s0)
-/var/run/nsd\.ctl	-s	gen_context(system_u:object_r:nsd_var_run_t,s0)
+/run/nsd/nsd\.pid	--	gen_context(system_u:object_r:nsd_var_run_t,s0)
+/run/nsd/nsd\.ctl	-s	gen_context(system_u:object_r:nsd_var_run_t,s0)
```

Comment:
The legacy /var/run path needs to be used because of equivalency rules. I think the parent directory should be labeled, too, or perhaps by one rule. File transition for the directory is not needed?